share/man/man3/sysexits.3

```diff
@@ -131,12 +132,23 @@ parenthesis for easy reference.
 .Sh HISTORY
 The
 .Nm
-file appeared somewhere after
-.Bx 4.3 .
+file first appeared in
+.Bx 4.0
+for use by delivermail, later renamed to
+.Xr sendmail 8 .
+There were sweeps in the tree in the 90s to introduce this,
+and then backout sweeps in the early 2000s.
+.Nm
+was officially deprecated in the base system in
+.Fx 8.0
+and is retained only for backwards compatibility.
```

Comment:
I'd use .Bx 4 (if it will let you) because it wasn't 4.0BSD, but 4BSD.

Also, I'd end it there, especially with the suggestion I made above.